.travis.yml

```diff
@@ -20,6 +20,24 @@ matrix:
       python: 3.5
     - os: linux
       python: 2.7
+    - os: linux
+      if: type = cron
```

Comment:
Can you please comment out this line (and the same line in the other integration tests jobs) temporarily so as to test that those integration tests pass in this PR before merging it in master?